shame_script.py

```diff
@@ -21,15 +23,18 @@
 
 TODOIST_API = "https://api.todoist.com/rest/v2/tasks"
 
+SCHEDULED_POST_TIME = time(hour=2)
```

Comment:
might be nice to make this configurable in the future
should we specify this is UTC time? SCHEDULED_POST_TIME_UTC maybe?